### DISCUSSION TITLE
How to redirect each container to http://ip-local/app?

### DISCUSSION THREAD

**Discussion by raphaeldallorto** [nginx-proxy/nginx-proxy#2198] [Q&A]
How can I configure docker compose file to redirect each container of each application to a URL like http://192.169.1.1/app and other to http://192.169.1.1/app2 and other to http://192.169.1.1/app3?

**Comment by cansurmeli** [comment]
What you need is `Path-based routing`. Check README for that.

**Reply by raphaeldallorto** [reply]
Hi, cansurmeli! Thank you for the answer.

I'm having a hard time understanding how to do this.
I'll give an example:
- The docker-compose.yml file which contains all services including nginx-proxy in /opt.
- The application is in the following path: /opt/app/frontend and /opt/app/backend

When I set VIRTUAL_HOST: 192.168.1.1 and VIRTUAL_PORT: 80 and try to access http://192.168.1.1 it works.

When I add VIRTUAL_PATH: /app build and up the containers and try to access http://192.168.1.1/app it doesn't work.
